rootfs-builder/README.md

```diff
@@ -121,6 +113,16 @@ containing the following expected files:
 
   Path: `rootfs-builder/<distro>/config.sh`.
 
+- (OPTIONAL) A `bash(1)` script called `build_rootfs.sh`
```

Comment:
Shouldn't this match the value of the `LIB_SH` variable? The scripts seem to still be looking for `rootfs_lib.sh`.

Also, note that all the `config.sh` files, the `Makefile` and `config_template.sh` need to be updated if the name has changed.

I'd consider:

- removing all the boiler-plate comments in all the `config.sh` files as it's documented here in the `README`.
- removing the the text from `config_template.sh` and referring the user to the `README` for more details.

That way, there is only 1 place which documents the files.